components/authz-service/engine/opa/policy/authz_v2.rego

```diff
@@ -73,9 +73,13 @@ match[[effect, pol_id, statement_id]] {
 	has_action[[pol_id, statement_id]]
 }
 
-allow = match[["allow", _, _]]
```

Comment:
Necessary change to make not inlining work. Also didn't work for non-partial eval. (See the OPA issue linked).